### ISSUE TITLE
mixed logging, one looks like logrus one looks like glog

### ISSUE THREAD

**Issue by chuckha** [kubernetes-sigs/cluster-api-provider-aws#324]
/kind bug

**What did you expect to happen:**
I expected our logs to be consistent in formatting:

```
{"level":"info","ts":1540584105.068694,"logger":"kubebuilder.controller","caller":"controller/controller.go:120","msg":"Starting EventSource","Controller":"machine-controller","Source":{"Type":{"metadata":{"creationTimestamp":null},"spec":{"metadata":{"creationTimestamp":null},"providerConfig":{},"versions":{"kubelet":""}},"status":{}}}}
{"level":"info","ts":1540584105.069299,"logger":"kubebuilder.controller","caller":"controller/controller.go:120","msg":"Starting EventSource","Controller":"cluster-controller","Source":{"Type":{"metadata":{"creationTimestamp":null},"spec":{"clusterNetwork":{"services":{"cidrBlocks":null},"pods":{"cidrBlocks":null},"serviceDomain":""},"providerConfig":{}},"status":{}}}}
{"level":"info","ts":1540584105.1697717,"logger":"kubebuilder.controller","caller":"controller/controller.go:134","msg":"Starting Controller","Controller":"cluster-controller"}
{"level":"info","ts":1540584105.1698842,"logger":"kubebuilder.controller","caller":"controller/controller.go:134","msg":"Starting Controller","Controller":"machine-controller"}
{"level":"info","ts":1540584105.2700703,"logger":"kubebuilder.controller","caller":"controller/controller.go:153","msg":"Starting workers","Controller":"machine-controller","WorkerCount":1}
I1026 20:01:45.270177       1 controller.go:113] Running reconcile Machine for aws-controlplane-4hqnj
I1026 20:01:45.270284       1 actuator.go:277] Checking if machine aws-controlplane-4hqnj for cluster test1 exists
I1026 20:01:45.270418       1 controller.go:173] Reconciling machine object aws-controlplane-4hqnj triggers idempotent create.
```

**Anything else you would like to add:**
This looks like logrus in one case and glog in another. Should we standardize on one?

**Environment:**

- Cluster-api-provider-aws version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

**Comment by randomvariable** [comment]
It's zap, which kubebuilder gives you by default, but cluster api hasn't fully migrated over either. 

Was mentioned in cluster api meeting on Wednesday. @justinsb and @dims are looking into it I think.

**Comment by vincepri** [comment]
Fixed in #377 
/close
